frontend/server/src/main/java/org/pytorch/serve/job/Job.java

```diff
@@ -11,7 +11,7 @@ public abstract class Job implements Prioritisable {
     private String modelVersion;
     private WorkerCommands cmd; // Else its data msg or inf requests
     private RequestInput input;
-    private int priority;
+    private String priority;
```

Comment:
Maybe we can use an [enum](https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html) instead? Then we could also validate the input with the `valueOf` factory method - although this one is case sensitive, so if we don't want that, some additional code would be necessary.
And we could throw an exception if the value is incorrect.

This way we have better type safety in code working with the priority, especially if we decide to add more priorities later or change some names.

Reply:
Great suggestion, will do.